Android/app/src/main/java/org/opendroneid/android/app/AircraftMapView.java

```diff
@@ -254,7 +254,6 @@ public void onActivityCreated(Bundle bundle) {
     public void onMapReady(@NonNull GoogleMap googleMap) {
         if (getActivity() == null)
             return;
-
```

Comment:
You have unnecessary white-space changes in multiple places in this PR. It would be nice if you could get rid of those.

(I sense that you are having a bit of struggles with git and these various white-space changes are probably a side-effect of your attempts to get your master branch up-to-date with the main repository's master. I would suggest you study a bit how to create a new branch for new development and then try to have your new branch always start from the latest HEAD of the main repository's master. It then becomes easier to keep your forks master branch in synch with the main repository's master).